Formula/osc-cli.rb

```diff
@@ -0,0 +1,26 @@
+class OscCli < Formula
+  include Language::Python::Virtualenv
+
+  desc "Official Outscale CLI providing connectors to Outscale API"
+  homepage "https://github.com/outscale/osc-cli"
+  url "https://github.com/outscale/osc-cli/releases/download/v1.6.0/osc-sdk-1.6.tar.gz"
+  sha256 "6319af4ee3f2effd991e8bd0fcd2b49914cf200915a7b1f606957db8eeb14ed8"
+  license "BSD-3-Clause"
+
+  depends_on "python@3.9"
+
+  def install
+    venv = virtualenv_create(libexec, "python3")
+
+    system libexec/"bin/pip", "install", "-v", "fire",
+                                  "--ignore-installed", buildpath
+    system libexec/"bin/pip", "uninstall", "-y", "osc-sdk"
+    venv.pip_install_and_link buildpath
+  end
+
+  test do
+    # we test the help wich is printed in stderr :(
+    str = shell_output("#{bin}/osc-cli help 2>&1 >/dev/null ; true")
```

Comment:
We don't want to do `; true`. Instead, pass the expected exit code as the second argument of `shell_output`.